jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/manageWebServer.feature

```diff
@@ -14,7 +14,7 @@ Scenario: Add Web Server
     When I click the add web server button
     And I see the web server add dialog
     And I fill in the "Web Server Name" field with "WEBSERVER_X"
-    And I fill in the "Host Name" field with "localhost"
+    And I fill in the "Host Name" field with "host1"
```

Comment:
In looking at this code now, I wonder: Is there a way to make these tests in the feature more readable? It feels like it defeats the purpose to have the value in the test point to a key in a property file some where. 
Is it possible to dynamically generate the feature from the property file, and then run the test from that generated feature. Then the test would be more readable, and the code base wouldn't require all the checks when reading a property.

Reply:
unfortunately in most of the cases...like this we need to have correct hostnames, and for security reasons we cannot have those in open source